profiles/common/modules/features/cce_basic_config/cce_basic_config.context.inc

```diff
@@ -32,12 +32,6 @@ function cce_basic_config_context_default_contexts() {
           'region' => 'tools',
           'weight' => 1,
         ),
-        'create-content-button' => array(
-          'module' => 'multisite_create_button',
```

Comment:
Are we not suppose to add this block on ec_resp as well? As it is it will only be enabled on ec_europa

Reply:
As discussed, this block is added by the multisite create button module and is now useless